packages/fx-core/tests/plugins/resource/dotnet/unit/deploy.test.ts

```diff
@@ -8,25 +8,24 @@ import fs from "fs-extra";
 import { PluginContext } from "@microsoft/teamsfx-api";
 import chaiAsPromised from "chai-as-promised";
 
-import { FrontendPlugin as BlazorPlugin } from "../../../../../src/plugins/resource/frontend";
+import { FrontendPlugin as WebappPlugin } from "../../../../../src/plugins/resource/frontend";
```

Comment:
I am a little confused on is it WebappPlugin or dotnetPlugin?

Reply:
I would like to distinguish dotnet plugin and webapp plugin. The former is for scaffolding and the latter is for provisioning and deploying.